ethapi/api.go

```diff
@@ -154,6 +154,13 @@ func (s *PublicEthereumAPI) EffectiveBaseFee(ctx context.Context) *hexutil.Big {
 	return (*hexutil.Big)(s.b.EffectiveMinGasPrice(ctx))
 }
 
+func (s *PublicEthereumAPI) BlobBaseFee(ctx context.Context) *hexutil.Big {
```

Comment:
Is there a way to query this as part of the Blog-Base-Fee integration test? If so, this might be a task for a follow-up PR, linking these changes up. If we ever implement blobs, we would be reminded to change this function as well.

Reply:
Actually I added comment to integration test PR, how to do it 
https://github.com/Fantom-foundation/Sonic/pull/255#issuecomment-2437482247